apps/climatemappedafrica/src/components/DataIndicators/DataIndicators.js

```diff
@@ -0,0 +1,214 @@
+import {
+  Box,
+  ClickAwayListener,
+  Grid,
+  Dialog,
+  Slide,
+  useMediaQuery,
+} from "@mui/material";
+import { useTheme } from "@mui/material/styles";
+import PropTypes from "prop-types";
+import React, { useState } from "react";
+
+import Icon from "./Icon";
+import IndicatorPanel from "./IndicatorPanel";
+
+import bg from "@/climatemappedafrica/assets/images/Mask Group 8.png";
+import Image from "@/climatemappedafrica/components/Image";
+import RichHeader from "@/climatemappedafrica/components/RichHeader";
+
+const Transition = React.forwardRef(function Transition(props, ref) {
+  return <Slide direction="left" timeout={300} ref={ref} {...props} />;
+});
+
+function DataIndicators({ indicators, title }) {
+  const theme = useTheme();
+  const isDesktop = useMediaQuery(theme.breakpoints.up("lg"));
```

Comment:
Most of the time, you don't need `theme`:
```suggestion
  const isDesktop = useMediaQuery((theme) => theme.breakpoints.up("lg"));
```